scripts/utils/create_package_template.py

```diff
@@ -27,7 +27,7 @@
     "Hex Editors",
     "Java",
     "Javascript",
-    "Net",
+    "Networking",
```

Comment:
are these the final categories? They are different from https://github.com/mandiant/VM-Packages/issues/38#issuecomment-1313874710

Note that modifying the categories also needs to update our current GH actions and the packages that are using the old categories (and maybe somewhere else?).